src/controllers/dlgprefcontrollerdlg.ui

```diff
@@ -318,7 +318,7 @@
              </sizepolicy>
             </property>
             <property name="text">
-             <string>Script Files:</string>
+             <string>Preset Files:</string>
```

Comment:
IIRC we wanted to get rid of the "Preset" term and use "Mapping" instead.

Reply:
okay, will do that for all user-visible strings.
Also I found some places were my changes can be improved.